geoana/gravity.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+from scipy.constants import G
+
+
+class GravityPointMass:
```

Comment:
Let's just name it `PointMass`, it already lives in the gravity module after all